modules/tinymce/CHANGELOG.md

```diff
@@ -59,6 +59,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `media` plugin no longer treats `iframe`, `video`, `audio` or `object` elements as "special" and will validate the contents against the schema #TINY-8382
 - Renamed the `getShortEndedElements` Schema API to `getVoidElements` #TINY-8344
 - Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- The `editor.editorCommands.queryCommandSupported` will no longer fallback to the browsers native command #TINY-7829
```

Comment:
It's probably best to put the word "function" or "API" in these lines, e.g. `the <x> function will...`

Reply:
I have wondered a bit about the word API to me an API is more than just a function it's kind of the whole thing. Like the tinymce editor has a selection API I think some times we use API for single functions what is the rule here exactly. I've used both my self. But it becomes really fuzzy if you have a function that only does one thing like the select element api like editor.selection.select() is that an API or a function?